Gemfile

```diff
@@ -4,7 +4,7 @@ source "https://rubygems.org"
 
 ruby RUBY_VERSION
 
-DECIDIM_VERSION = "0.23.2"
+DECIDIM_VERSION = { git: "https://github.com/decidim/decidim.git", branch: "release/0.23-stable" }.freeze
```

Comment:
is there a chance we could relax the minor version with `~>0.23.2` so that we can still rely on Rubygems and not Github?

Reply:
if you prefer, yes. However i found this more versatile. Do you want me to change it?